.github/workflows/ci.yml

```diff
@@ -0,0 +1,257 @@
+---
+name: Continuous Integration tests
+
+on: [pull_request]
+
+jobs:
+  main_tests:
+    name: Main tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9']
+        # ['2.7', '3.6', '3.7', '3.8', '3.9', '3.10.0-rc.1', 'pypy-2.7', 'pypy-3.6', 'pypy-3.7-nightly']
+        include:
+          - os: ubuntu-latest
+            python-version: '3.6'
+          - os: ubuntu-latest
+            python-version: '3.7'
+          - os: ubuntu-latest
+            python-version: '3.8'
+          ## Expected failure
+          # - os: ubuntu-latest
+          #   python-version: '3.10.0-rc.1'
+          ## Expected failure
+          # - os: ubuntu-latest
+          #   python-version: 'pypy-3.6'
```

Comment:
See: https://github.com/dralley/hpy/runs/3295502351?check_suite_focus=true#step:4:797

Reply:
yes, we can't/shouldn't run tests on pypy.
PyPy releases its own bundled version of hpy and take care of running its own test suite, so here we just need to test cpython

Reply:
I'll drop it from the PR then.